BUILDING.md

```diff
@@ -48,3 +50,15 @@ To build without executing tests:
 To find the Java binary in your environment, run the appropriate command for your operating system:
 * Linux/macOS: `which java`
 * Windows: `for %i in (java.exe) do @echo. %~$PATH:i`
+
+## Running tests with TinkerPop 3.2.3
+
+Several TinkerPop tests require commit `7aa9782`, which was merged after the release of TinkerPop 3.2.3. Tests requiring this commit include BulkDumperVertexProgramTest and PeerPressureTest. In order to run these tests TinkerPop must be locally patched and built as shown below.
+
+```bash
+git clone https://github.com/apache/tinkerpop.git
+cd tinkerpop
+git checkout 3.2.3
+git cherry-pick 7aa9782
+mvn clean install -DskipTests -Drat.skip=true
+```
```

Comment:
Can you create an issue to track updating this part of the docs after we move to 3.2.4? Also please note here that the dependency on 7aa9782 is only a compile-test dependency.